ricecooker/classes/nodes.py

```diff
@@ -37,17 +37,21 @@ def guess_content_kind(files, questions=None):
 
 class Node(object):
     """ Node: model to represent all nodes in the tree """
-    def __init__(self, title, description=None, thumbnail=None):
+    def __init__(self, title, description=None, thumbnail=None, license=None, copyright_holder=None):
```

Comment:
I think we do need to make license a required field, to ensure they're being specified.